hw06-atm/src/main/java/Bankomat.java

```diff
@@ -0,0 +1,9 @@
+import java.util.HashMap;
+
+public interface Bankomat {
+    void addMoney(HashMap<CashType, Integer> money);
+
+    HashMap<CashType, Integer> getMoney(int money) throws Exception;
```

Comment:
```suggestion
    void addMoney(Map<CashType, Integer> money);

    Map<CashType, Integer> getMoney(int money) throws Exception;
```

Параметры и возвращаемые типы определяйте как интерфейсы. Не зря же вы сделали интерфейс `Bankomat`.